File: FinesTracker/ClientApp/src/app/Models/FinePost.ts
import {User} from './User';
import {Fine} from './Fine';
import {Vote} from './Vote';

export class FinePost {
  public finePostId!: number;
  public fineId!: number;
  public finerId!: number;
  public fineeId!: number;

  public fine!: Fine;
  public finer!: User;
  public finee!: User;

  public votes!: Vote[];
}

File: Domain/FinePost.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace Domain
{
    public class FinePost
    {
        public int FinePostId { get; set; }
        public int FineId { get; set; }
        public int FinerId { get; set; }
        public int FineeId { get; set; }

        public virtual Fines Fine { get; set; }
        public virtual User Finee { get; set; }
        public virtual User Finer { get; set; }
        public virtual IEnumerable<Vote> Votes { get; set; }
    }
}
